doc/backend/norms-api-v1.yaml

```diff
@@ -204,6 +204,30 @@ paths:
               schema:
                 type: string
 
+  /renders:
+    post:
+      summary: Create and return a HTML render of a provided law
+      parameters:
+        - in: query
+          name: show_metadata
+          schema:
+            type: boolean
+            default: false
+          description: Should the metadata table be included?
+      requestBody:
+        description: The XML of the law
+        required: true
+        content:
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/LegalDocML.de'
+      responses:
+        '200':
+          description: HTML render of the given law
```

Comment:
If we change the terminology above ("rendering", "LDML.de XML"), then also here.